src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/VsProjectJsonNuGetProject.cs

```diff
@@ -76,7 +76,15 @@ internal class VsProjectJsonNuGetProject : ProjectJsonNuGetProject
                         ? new Version(platformMinVersionString)
                         : null;
 
-                    if (platformMinVersion != null && jsonTargetFramework.Version != platformMinVersion)
+                    var targetFrameworkMonikerString = await _vsProjectAdapter
+                        .BuildProperties
+                        .GetPropertyValueAsync(ProjectBuildProperties.TargetFrameworkMoniker);
+
+                    var targetFrameworkMoniker = !string.IsNullOrWhiteSpace(targetFrameworkMonikerString)
+                        ? NuGetFramework.Parse(targetFrameworkMonikerString)
+                        : null;
+
+                    if (platformMinVersion != null && jsonTargetFramework.Version != platformMinVersion && FrameworkConstants.CommonFrameworks.NetCore50 == targetFrameworkMoniker)
```

Comment:
Did you confirmed that all c# UAP based projects will have netcore50 as TFM? like how about PCL (with uap, desktop, netstandard)? Just confirm